### DISCUSSION TITLE
Question about license

### DISCUSSION THREAD

**Discussion by prasanna4742** [SigNoz/signoz#2684] [Q&A] [answered]
Hi,

If I want to use community open-source edition in production - do I need to clone the repo, remove ee directory, build and then use that binary?
Or 
I can simply use the docker image provided by signoz, and enterprise features would be locked somehow?
(Point is, I want to use community edition in production, do not want to end-up violating license just becasue I'm using direct docker image.)

Thanks,
Prasanna

**Comment by ankitnayan** [comment, marked as answer]
>I can simply use the docker image provided by signoz, and enterprise features would be locked somehow?

This is correct

**Reply by prasanna4742** [reply]
Thanks for prompt reply Ankit.
